src/aliceVision/feature/sift/ImageDescriber_SIFT_popSIFT.hpp

```diff
@@ -127,8 +119,6 @@ class ImageDescriber_SIFT_popSIFT : public ImageDescriber
 
 private:
 
-  void resetConfiguration();
```

Comment:
It would be better to keep the private function `resetConfiguration` instead of having all this logic inside the `describe` method.